include/iso15118/d20/control_event.hpp

```diff
@@ -51,7 +54,13 @@ class StopCharging {
     bool stop;
 };
 
-using ControlEvent =
-    std::variant<CableCheckFinished, PresentVoltageCurrent, AuthorizationResponse, StopCharging, DcTransferLimits>;
+struct UpdateDynamicModeParameters {
+    std::optional<std::time_t> departure_time;
+    std::optional<std::uint8_t> target_soc;
+    std::optional<std::uint8_t> min_soc;
+};
+
+using ControlEvent = std::variant<CableCheckFinished, PresentVoltageCurrent, AuthorizationResponse, StopCharging,
```

Comment:
What I mentioned above for the feedback, has been basically implemented already for the other direction with the `ControlEvent`

Reply:
:+1: